tests/test_spstats.py

```diff
@@ -156,6 +156,46 @@ def test_neff_estimation(self):
 
                 assert np.abs(neff_circ_exact-neff_circ_numer) < 0.001
 
+class TestSubSampling:
+
+    def test_circular_masking(self):
+        """Test that the circular masking works as intended"""
+
+        # using default (center should be [2,2], radius 2)
+        circ = xdem.spstats.create_circular_mask(5,5)
+        circ2 = xdem.spstats.create_circular_mask(5,5,center=[2,2],radius=2)
+
+        assert np.array_equal(circ,circ2)
```

Comment:
You could also test that it is exactly equal to the matrix you expect i.e. 0 and 1 with the correct shape. This would be a more rigorous test!

Reply:
Added